context-core/src/main/java/de/atb/context/common/util/BusinessCase.java

```diff
@@ -35,7 +35,16 @@
 	private String id;
 	private String url;
 
-	public static void getInstance() {
+    public BusinessCase() {
+        if (settings.get(NS_DUMMY_ID) == null) {
+            settings.put(NS_DUMMY_ID, new BusinessCase(NS_DUMMY_ID, NS_DUMMY_URL));
+        }
+        if (settings.get(NS_BASE_ID) == null) {
+            settings.put(NS_BASE_ID, new BusinessCase(NS_BASE_ID, NS_BASE_URL));
+        }
+    }
+
+    public static void getInstance() {
```

Comment:
Maybe rename this method? `getInstance` would imply that some kind of instance is returned, not `void`. It might also be easily confused with the other `getInstance` method, which actually returns a `BusinessCase`

Reply:
I would suggest to not do this change in this Merge Request, but consider it for improvement in a later dev branch

Reply:
Added TODO in the Code